src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Tools.Add.PackageReference
+{
+    internal class LocalizableStrings
```

Comment:
This is going to be problematic, as we are in LOC freeze.

@srivatsn Would it be possible to take these strings at this point? Any way we can send the loc folks an updated set of xlf files?